config/abilities/provider_admin.rb

```diff
@@ -76,6 +76,8 @@
       service.account_id == user.account_id && can?(:manage, :multiple_services) and not service.default_or_last?
     end
 
+    can :edit, BackendApi
```

Comment:
Shouldn't this be only if the rolling update enables it?

Reply:
because right now i think that any provider admin without the rolling update for `api_as_product` would be able to use the controller in this PR.